### PR TITLE
Welcome Prompt

### LINKED ISSUE #7: 7. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works.
AC:
      * The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item
      Notes:
      * The wireframe is a little short on the "welcoming" part of the story, but otherwise is functionally complete for the purposes of the AC. https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/list-empty.png

### PR DESCRIPTION
**Pull request by sanrafa**
## Description

We added a div element which renders when the user has no items on their list, prompting them to click the button and navigate to the AddItemView. 

Home navigation was removed from the List view. Opened this PR #26 to add it back as a persistent nav.

## Related Issue

Closes #7 

## Acceptance Criteria

AC:
* The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item
Notes:
* The wireframe is a little short on the "welcoming" part of the story, but otherwise is functionally complete for the purposes of the AC. https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/list-empty.png

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Walk-Through

https://user-images.githubusercontent.com/76231436/152038556-f265f909-b7d9-4b8e-b985-21f3f7e3305c.mp4

### CODE REVIEW

**Review by danainjax — APPROVED** [review]
Welcome prompt achieved! I am seeing one thing, though. Looks like The Navigate back to Home was deleted by mistake in the ListView?

**Comment by sanrafa** [comment]
> Welcome prompt achieved! I am seeing one thing, though. Looks like The Navigate back to Home was deleted by mistake in the ListView?

Appreciate the call-out, Dana! We originally removed it because it seemed redundant, but perhaps a good compromise would be a persistent navigation element at the top? Created a PR here: #26

**Review by vividavila98 — APPROVED** [review]
Everything works as expected, great job! :)

**Review by yenly — APPROVED** [review]
Nice write up and I appreciate the video walk though.

File: src/components/List.jsx [hunk 1]
```diff
@@ -1,25 +1,34 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
 import db from '../lib/firebase';
 import { collection } from 'firebase/firestore';
 import { useCollection } from 'react-firebase-hooks/firestore';
 
 export const List = ({ token }) => {
+  let navigate = useNavigate();
```
Comment (tannaurus) [inline src/components/List.jsx:8]: You should only use `let` when you are reassigning the variable somewhere else. In this case `const` would be more appropriate

**Review by tannaurus — APPROVED** [review]

**Pull request merged**